roofit/batchcompute/CMakeLists.txt

```diff
@@ -0,0 +1,65 @@
+# Library which powers fast batch computations in Roofit.
+ROOT_STANDARD_LIBRARY_PACKAGE(RooBatchCompute
+  HEADERS
+    RooBatchCompute.h
+    BracketAdapter.h
+    RooSpan.h
+    RooVDTHeaders.h
+    RunContext.h
+  SOURCES
+    src/Initialisation.cxx
+    src/RunContext.cxx
+  DICTIONARY_OPTIONS
+    "-writeEmptyRootPCM"
+  #~  LINKDEF
+    #~  LinkDef1.h
+  DEPENDENCIES
+    Core
+    MathCore
+)
+
+if(vdt AND NOT builtin_vdt)
+  target_include_directories(RooBatchCompute PUBLIC ${VDT_INCLUDE_DIR})
+endif()
+
+if(CMAKE_CXX_COMPILER_ID STREQUAL "GNU")
+  # gcc has a bug in that it fails to put its internal -lgcc into the right place when linking.
+  # We need it to check cpu flags in src/Initialisation.cxx
+  # Here, we add an explicit link instruction according to the workaround posted here:
+  # https://bugs.launchpad.net/ubuntu/+source/gcc-5/+bug/1568899
+  target_link_libraries(RooBatchCompute PRIVATE -lgcc_s -lgcc)
+endif()
+
+
+############################################################################################################################################
+# Instantiations of the shared objects which provide the actual computation functions.
+
+# Generic implementation for CPUs that don't support vector instruction sets.
+ROOT_LINKER_LIBRARY(RooBatchCompute_GENERIC src/RooBatchCompute.cxx TYPE SHARED DEPENDENCIES RooFitCore RooBatchCompute)
```

Comment:
I have the same quiestion about naming for RooBatchCompute_GENERIC/RooBatchCompute_SSE4.1/RooBatchCompute_AVX and RooBatchCompute_AVX2 libraries, should they start with prefix ROOT? (cc @Axel-Naumann )

Reply:
These are roofit libraries and we should keep the same convention ans tart their names with `Roo`. So I think th ecurrent name is fine!